docs/releases/v0.3.0-release-notes.md

```
@@ -0,0 +1,11 @@
+# dev.to Enhanced v0.3.0 - Release Notes
+
+## 🚀 What's New
+
+### Performance Improvements
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)